ospd_openvas/wrapper.py

```diff
@@ -913,10 +913,19 @@ def stop_scan_cleanup(
                     logger.debug(
                         'Process with pid %s already stopped', ovas_pid
                     )
-                self.openvas_db.release_db(current_kbi)
                 if parent:
-                    parent.send_signal(signal.SIGUSR2)
+                    cmd = ['sudo', 'openvas', '--scan-stop', scan_id]
+
+                    try:
+                        result = subprocess.Popen(cmd, shell=False)
+                    except OSError:
+                        # the command is not available
```

Comment:
the log message should also include the reason from the error

Reply:
It could also make sense to include the requirement for a full / correct sudo setup in the installation docs.

Reply:
In future, if sudo fails, it will call openvas without sudo.

Reply:
Which probably won't solve the issue if e.g. openvas is started as root (but without sudo) and ospd-openvas is started as non-root user.